cogs/help.py

```diff
@@ -0,0 +1,238 @@
+        """Send help for a cog."""
+        ctx = self.context
+        prefix = ctx.prefix
+        embed = discord.Embed(
+            title=cog.qualified_name,
+            description=textwrap.dedent(
+                f"""
+                Help syntax : `<Required argument>`. `[Optional argument]`
+                Command prefix: `{self.prefix}`
+                {cog.description}
+                """
+            ),
+            color=discord.Color.blue(),
+        )
+        embed.set_author(
+            name=str(ctx.message.author),
+            icon_url=str(ctx.message.author.avatar_url),
+        )
+        embed.set_thumbnail(url=str(ctx.bot.user.avatar_url))
+        for command in await self.filter_commands(cog.get_commands()):
+            embed.add_field(
+                name=f"{prefix}{self.get_command_signature(command)}",
+                value=command.help,
+                inline=False,
+            )
+        embed.set_footer(
+            text=f"Are you interested in {cog.qualified_name}?",
+            icon_url=str(ctx.bot.user.avatar_url),
+        )
+        await ctx.send(embed=embed)
+
+    async def send_command_help(self, command: commands.Command) -> None:
+        """Send help for a command."""
+        ctx = self.context
+        prefix = ctx.prefix
+        embed = discord.Embed(
+            title=f"{prefix}{self.get_command_signature(command)}",
+            description=textwrap.dedent(
+                f"""
+                Help syntax : `<Required arguments`.
+                `[Optional arguments]`\n{command.help}
+                """
+            ),
+            color=discord.Color.blue(),
+        )
+        if command.aliases:
+            embed.add_field(name="Aliases :", value="\n".join(command.aliases))
+        embed.set_author(
+            name=str(ctx.message.author),
+            icon_url=str(ctx.message.author.avatar_url),
+        )
+        embed.set_thumbnail(url=str(ctx.bot.user.avatar_url))
+        if command.hidden:
+            embed.set_footer(
+                text=f"Wow, you found {command.name}!",
+                icon_url=str(ctx.bot.user.avatar_url),
+            )
+        else:
+            embed.set_footer(
+                text=f"Are you interested in {command.name}?",
+                icon_url=str(ctx.bot.user.avatar_url),
+            )
+        await ctx.send(embed=embed)
+
+    async def send_group_help(self, group: commands.Group) -> None:
+        """Send help for a group."""
+        ctx = self.context
+        prefix = ctx.prefix
+        embed = discord.Embed(
+            title=textwrap.dedent(
+                f"""
+                Help for group {prefix}
+                {self.get_command_signature(group)}
+                """
+            ),
+            description=textwrap.dedent(
+                f"""
+                Help syntax : `<Required arguments>`.
+                `[Optional arguments]`\n{group.help}
```

Comment:
We usually avoid using `\n` in textwrap.dedent multi-line strings, since these already are multiline there is no reason to use `\n`
```suggestion
                `[Optional arguments]`
                {group.help}
```

Reply:
Okay done